src/shared/sam/debugger/goSamDebug.ts

```diff
@@ -32,10 +32,9 @@ export async function invokeGoLambda(ctx: ExtContext, config: GoDebugConfigurati
     config.onWillAttachDebugger = waitForDelve
 
     if (!config.noDebug && !(await installDebugger(config.debuggerPath!))) {
-        getLogger('channel').warn(
-            localize('AWS.sam.debugger.godelve.failed', 'Failed to install Delve. Code will execute without debugging.')
+        throw new Error(
+            localize('AWS.sam.debugger.godelve.failed', 'Failed to install Delve. Please see logs for details.')
```

Comment:
avoid "please" in messages. Advantages are:

- No one will be offended :) 
- It saves space
- It helps the customer by drawing more attention to the actual, important contents of the message

Reply:
Also any time we have "see logs" we might as well use `showErrorWithLogs()` (hopefully whatever catches this error does that...)

Reply:
Throwing an error here causes a VS Code dialog to pop up asking if the user wants to view the launch config. I'll change it to use `showErrorWithLogs` instead since the failure is unrelated to the config.

Reply:
I also clarified that the Delve debugger failed to install for the lambda container. A user might be confused why the extension is trying to install Delve even though they have the Delve binary in their GOPATH.